### ISSUE TITLE
pascal dataset to train a specific class

### ISSUE THREAD

**Issue by maylad31** [pjreddie/darknet#1205]
Hi,thanks for the code.I would like to know how do I train pascal dataset again for specific class like person. I have modified cfg file and modified filters,width and height. I have created a python script that removes data of all other classes except person from .txt file and changes class id of person from 14 to 0. Created names and data files. When I begin training loss decreases and goes to about 1 but then it suddenly increases and becomes nan. Please help.Thanks.

**Comment by fengxiuyaun** [comment]
maybe reduce learning rate
